tests/utils/test_ensembl_rest_clients.py

```diff
@@ -4,53 +4,33 @@
 from requests.exceptions import MissingSchema
 from requests.models import Response
 
-from scout.utils.ensembl_rest_clients import RESTAPI_37
+from scout.utils.ensembl_rest_clients import RESTAPI_URL
 
 
 @responses.activate
-def test_liftover(ensembl_rest_client_37, ensembl_liftover_response):
+def test_liftover(ensembl_rest_client, ensembl_liftover_response):
     """Test send request for coordinates liftover"""
     # GIVEN a patched response from Ensembl
-    url = f"{RESTAPI_37}/map/human/GRCh37/X:1000000..1000100/GRCh38?content-type=application/json"
+    url = f"{RESTAPI_URL}/map/human/GRCh37/X:1000000..1000100/GRCh38?content-type=application/json"
     responses.add(
         responses.GET,
         url,
         json=ensembl_liftover_response,
         status=200,
     )
-    client = ensembl_rest_client_37
+    client = ensembl_rest_client
     # WHEN sending the liftover request the function should return a mapped locus
     mapped_coords = client.liftover("37", "X", 1000000, 1000100)
     assert mapped_coords[0]["mapped"]
 
 
-@responses.activate
-def test_send_gene_request(ensembl_gene_response, ensembl_rest_client_37):
-    """Test send request with correct params and endpoint"""
-    url = f"{RESTAPI_37}/overlap/id/ENSG00000103591?feature=gene"
```

Comment:
This overlap functionality is not even used in Scout so the test is pointless

Reply:
🥇  We can add back if use it again.